aries-site/src/pages/templates/forms.mdx

```diff
@@ -23,105 +41,72 @@ import { Example } from '../../layouts';
+- List only the most common field requirements below the relevant field. 
+- Use asterisks for required fields when the form has more than two required fields and/or has optional fields. 
+- Asterisks for required fields should be the same color as label field. 
+- If a field is optional, write "Optional" next to the form field label.
 
-Button labels should be specific to the action of the button. For example, a [sign-up form](#sign-up) may have a submit button with label "Sign Up". Alternately, a [password change form](#change-password) may have a button with label "Update Password".
+**Format and fields**
+- User-entered input must be encoded to prevent [SQL injection](https://www.w3schools.com/sql/sql_injection.asp) and [XSS attacks](https://en.wikipedia.org/wiki/Cross-site_scripting).
+- Wrap all input elements within a FormField `<FormField label="Username"><TextInput /></FormField>`  
+- Reveal fields progressively so the user is not overwhelmed.
+- Write in sentence case (capitalize the first letter and all following letters should be lowercase).  
+- Fill in default values for as many fields as possible. 
+- Use clear, concise title labels for each field. 
+- Write helpful placeholder text inside the fields. 
+- If the user would likely have doubts, include helpful text below the field, such as [password requirements](#signup) or [additional resources](#customize) on a field's context.
+- Place the submit, reset, and cancel buttons at the bottom left of the form. [See responsive button behavior](https://design-system.hpe.design/components/button?q=button#responsiveness). 
+- Use specific labels on buttons that tell the user what action to expect. 
 
-Button label specificity provides users confidence in the action they are performing.
+**Actions after the user submits information**
+- Provide feedback when the form is submitted successfully (e.g. notification, success page, green check mark, etc.). 
+- [In the case of an error](#requiredoptional), red highlight the field where the error occurred. 
+- List below the button all relevant errors and the corrective action the user must take. 
+- Allow the user to continue navigating through the application immediately.
 
-### User-Entered Input
 
-There should be few restrictions on user-entered input.
+## Don'ts
 
-- Resource naming should allow for spaces, punctional characters, unicode characters.
-  - Examples: Björk Guðmundsdóttir, 联想, John Q. Public, etc.
-- User-entered input needs to be encoded to prevent [SQL injection](https://www.w3schools.com/sql/sql_injection.asp) and [XSS attacks](https://en.wikipedia.org/wiki/Cross-site_scripting).
+**Required fields**
+- Use asterisks for sign up and sign in forms. 
+- Red asterisks stresses the user and actually decrease success in filling out the form. 
+- No need to list requirements up front that the user is unlikely to commit (ex. double hyphens, starting with a space, etc.). 
```

Comment:
What does this mean?

Reply:
Edgar mentioned this. Do not list rejection criteria on the form if the user is unlikely to type them, such as double hyphens, starting with a space, etc. If they happen to violate the requirements by typing a double hyphen or starting with a space, THEN you give them an error and list the infraction. Do you think the user would understand that?

Reply:
- List rejection criteria that the user is unlikely to commit (ex. double hyphens, starting with a space, etc.). Only when they commit the error do you alert them.